store/migration/dev/LATEST_DATA.sql

```diff
@@ -35,6 +35,7 @@ VALUES
 
 ALTER SEQUENCE environment_id_seq RESTART WITH 103;
 
+-- TODO(d): update policy data.
```

Comment:
This is dev, so if we don't do anything here, will we break the head?

Reply:
Yup, dev environment uses the new schema.